xml/yast2_ncurses.xml

```diff
@@ -442,11 +422,10 @@
       <term>interactive</term>
       <listitem>
        <para>
-        If you need to spend more time querying a module's settings, run the
-        <emphasis>interactive</emphasis> mode. The &yast; shell opens, where
-        you can enter all the module's commands without the <command>sudo yast
-        ...</command> prefix. To leave the interactive mode, enter
-        <literal>exit</literal>.
+	Enters the <emphasis>interactive</emphasis> mode that lets you run the
```

Comment:
What does 'them' refer to? Should it perhaps be 'commands' (plural) before? Also, for better readability, I would  split into two sentences: 'Enters the ```<emphasis>interactive</emphasis>``` mode. It lets you ...

Reply:
Indeed, command**s**